src/hotspot/share/classfile/symbolTable.cpp

```diff
@@ -87,7 +87,7 @@ static size_t _symbols_removed = 0;
 static size_t _symbols_counted = 0;
 static size_t _current_size = 0;
 
-static volatile size_t _items_count = 0;
+static volatile int    _items_count = 0;
```

Comment:
Shouldn't there be a check somewhere that this value doesn't overflow?

Reply:
Yes, there should be here too.